README.md

```diff
@@ -35,7 +35,7 @@ Once you are familiar with these concepts you can start drilling down into
 patterns by any of the following approaches
 
  - Using difficulty tags, `Difficulty-Beginner`, `Difficulty-Intermediate` & `Difficulty-Expert`.
- - Using pattern categories, `Creational`, `Behavioral` and others.
+ - Using pattern categories, `Creational`, `Behavioral`,and others.
```

Comment:
Shouldn't there be a space after the comma?

Reply:
Indeed, a space is needed.